packages/spaces/test/commands/transfer.js

```diff
@@ -0,0 +1,53 @@
+'use strict'
+/* globals describe beforeEach it */
+
+const nock = require('nock')
+const cmd = require('../../commands/transfer')
+const expect = require('chai').expect
+const cli = require('heroku-cli-util')
+
+describe('spaces:transfer', function () {
+  beforeEach(() => cli.mockConsole())
+
+  it('yields success when the API succeeds', function () {
+    const space = 'dimension-c137'
+    const team = 'jerry'
+    const api = nock('https://api.heroku.com:443')
+      .post(`/spaces/${space}/transfer`, {
+        'new_owner': team
+      })
+      .reply(201,
+        {
+          'created_at': '2019-01-09T22:31:33Z',
+          'id': '5dd30c44-078f-4424-8585-cb98adf86723',
+          'name': space,
+          'organization': { 'id': '12cd6520-b000-4882-a655-8ae84a0132cb', 'name': team },
+          'team': { 'id': '12cd6520-b000-4882-a655-8ae84a0132cb', 'name': team },
+          'region': { 'id': '3544427c-5b3b-4e1e-b01a-b66362573b26', 'name': 'virginia' },
+          'shield': false,
+          'state': 'allocated',
+          'cidr': '10.0.0.0/16',
+          'data_cidr': '10.1.0.0/16',
+          'updated_at': '2019-07-16T10:19:10Z'
+        }
+      )
+    return cmd.run({ flags: { team, space } })
+      .then(() => expect(cli.stdout).to.equal(``))
```

Comment:
It seems like this should equal something other than a blank string?

Reply:
excellent catch; updated to look for `done` from `stderr` instead!